doc/stdenv/meta.chapter.md

```diff
@@ -244,6 +244,10 @@ Unfree package that cannot be redistributed. You can build it yourself, but you
 
 This package supplies unfree, redistributable firmware. This is a separate value from `unfree-redistributable` because not everybody cares whether firmware is free.
 
+## Categories {#sec-meta-categories}
+
+The categories of the package. A list of attributes from the attribute set defined in [`nixpkgs/lib/categories-set.nix`](https://github.com/NixOS/nixpkgs/blob/master/lib/categories-set.nix).
```

Comment:
```suggestion
The `meta.categories` attribute contains the categories of the package. A list of attributes from the attribute set defined in [`nixpkgs/lib/categories-set.nix`](https://github.com/NixOS/nixpkgs/blob/master/lib/categories-set.nix).
```

Reply:
I still think the attribute name should be made explicit like in other entries.

Reply:
Can you explain? I'm not understanding.

Reply:
Some sections are titled with _exact_ (including capitalisation) name of the meta attribute. Some sections include the full name in the text near the beginning. Some do both.

The proposed section here does neither, thus my suggestion to add `meta.categories` into the text of the section.

Reply:
Done.